src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $cardWidth;
+  line-height: $lineHeitghtBig;
```

Comment:
There is a typo in the variable name `$lineHeitghtBig`. It should be `$lineHeightBig` to follow the correct spelling of 'height'.